vmm-reference-interrupts.md

```diff
@@ -0,0 +1,113 @@
+# Interrupts (x86_64 architecture)
+* source file: vmm-reference/src/vm-vcpu-ref/src/x86_64/interrupts.rs 
+* Note: This document only covers the x86_64 architecture. For aarch64, the interrupt controller is called GIC (Generic Interrupt Controller). Refer to vmm-reference/src/vm-vcpu-ref/src/aarch64/interrupts.rs
+## Local Advanced Programmable Interrupt Controller (LAPIC)
+LAPIC is a memory-mapped device that is used to send interrupts to the VCPUs. It is used to send interrupts to the VCPUs and to receive interrupts from the VCPUs. It is also used to send inter-processor interrupts (IPIs) to the VCPUs.
+
+There are five delivery modes for interrupts:
```

Comment:
It will be good to know the source of this information. Example "According to the Section xyz from Intel Manual volume abc".